src/transform/ThemeTransform.css

```diff
@@ -35,17 +35,121 @@
   background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAAadEVYdFNvZnR3YXJlAFBhaW50Lk5FVCB2My41LjEwMPRyoQAAAGJJREFUKFN1jdENgCAMBYmJn47Bak7DZrhTpc/XIm34OAjXA4qIgHI/dSBbLGTcOKjBryFlinGmjDQGiOF0MQkxI3v5wq6L38qR7SnsAx8ul37igPjAd+o5Oz2MRA+xY4ZSXuaW6wYouOLpAAAAAElFTkSuQmCC);
```

Comment:
> may be just as likely to break elements that were intentionally styled differently

I think it's worth keeping the theme user story in mind - which is precisely about "overriding elements which were intentionally styled differently". The '[baseline](https://github.com/wikimedia/wikimedia-page-library/pull/77#discussion_r131254971)' approach is unfortunately the only way I know to robustly apply themes to inline styles. I've not seen any "break(ing) elements" with this approach, just the disappearance of all known issues :)

-----

Oh, side note - I just updated all of the `after` screenshots.

One more - I also added articles from OTRS article theming bug reports to the theme demo article list and confirmed reported issues are resolved with the PR.